stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Push.java

```diff
@@ -39,62 +36,32 @@ public void registeringDevice() {
                 }
             });
         }
-
-        /**
-         * @see <a href="https://getstream.io/chat/docs/push_android/?language=java#setting-up-notification-data-payload-at-stream-dashboard">Setting up notification</a>
-         */
-        public void setupNotifications() {
-            int notificationChannelId = R.string.stream_chat_notification_channel_id;
-            int notificationChannelName = R.string.stream_chat_notification_channel_name;
-            int smallIcon = R.drawable.stream_ic_notification;
-            String firebaseMessageIdKey = "message_id";
-            String firebaseMessageTextKey = "message_text";
-            String firebaseChannelIdKey = "channel_id";
-            String firebaseChannelTypeKey = "channel_type";
-            String firebaseChannelNameKey = "channel_name";
-            int errorCaseNotificationTitle = R.string.stream_chat_notification_title;
-            int errorCaseNotificationContent = R.string.stream_chat_notification_content;
-            boolean useProvidedFirebaseInstance = true;
-
-            NotificationConfig notificationsConfig = new NotificationConfig(
-                    notificationChannelId,
-                    notificationChannelName,
-                    smallIcon,
-                    firebaseMessageIdKey,
-                    firebaseMessageTextKey,
-                    firebaseChannelIdKey,
-                    firebaseChannelTypeKey,
-                    firebaseChannelNameKey,
-                    errorCaseNotificationTitle,
-                    errorCaseNotificationContent,
-                    useProvidedFirebaseInstance
-            );
-
-            MyNotificationHandler notificationHandler = new MyNotificationHandler(context, notificationsConfig);
-
-            new ChatClient.Builder("{{ api_key }}", context)
-                    .notifications(notificationHandler)
-                    .build();
-        }
     }
 
     /**
-     * @see <a href="https://getstream.io/chat/docs/android/push_android/?language=java#redirection-from-notification-to-app">Redirection from notification to app
-    </a>
+     * @see <a href="https://getstream.io/chat/docs/android/push_android/?language=java#redirection-from-notification-to-app">Redirection from notification to app</a>
+     *
+     * <pre>
```

Comment:
Every minor change requires overriding the receiver from SDK with a custom one with "io.getstream.chat.SHOW_NOTIFICATION" action